delfin/db/api.py

```diff
@@ -841,3 +841,491 @@ def failed_task_delete_by_storage(context, storage_id):
     does not exist.
     """
     return IMPL.failed_task_delete_by_storage(context, storage_id)
+
+
+def storage_initiators_create(context, values):
```

Comment:
Can we use just `initiators` instead of `storage_initiators` ?

Reply:
Updated